doc/apiv3-documentation.apib

```diff
@@ -2149,17 +2149,19 @@ Revisions are sets of updates to files in the context of repositories linked in
 |:----------------:| --------------------------------------------------------------------------------------------------| ------------- | ----------- | -------------------- |
 | self             | This revision                                                                                     | Revision      | not null    | READ                 |
 | project          | The project to which the revision belongs                                                         | Project       | not null    | READ                 |
-| author           | The user that added this revision, if the authorName was mapped to a user in OpenProject       | User          |             | READ                 |
+| author           | The user that added this revision, if the authorName was mapped to a user in OpenProject          | User          |             | READ                 |
+| showRevision     | A URL to the repository view (outside APIv3) showing this revision                                | String        | not null    |                      |
```

Comment:
I think I usually described the type of such links to be `-`. The meaning of the type is "what type does the resource referenced by this link have?"

The referenced resource is no APIv3 resource, therefore it has no real type.

At least that was my take on it ^^